aten/src/ATen/native/BatchLinearAlgebra.h

```diff
@@ -4,6 +4,16 @@
 #include <ATen/Config.h>
 #include <ATen/native/DispatchStub.h>
 
+#if AT_MKL_ENABLED()
```

Comment:
ILP64 is not specific to MKL, it's possible to build OpenBLAS with ILP64.

Reply:
Right. While slightly OT to this PR, I have inserted
```c++
#ifdef OPENBLAS_USE64BITINT
#define INT_T int64_t
```

However, I suspect that the support for building pytorch with OpenBLAS ILP64 model requires more than that. For instance, detecting the openblas64 library is not implemented in cmake/Modules/FindOpenBLAS.cmake.

Reply:
I wonder why PyTorch does not use standard CMake Modules.  Since CMake 3.22, it has a support of finding the specific LP64/IPL64 models (https://cmake.org/cmake/help/v3.22/module/FindBLAS.html)

Reply:
For no specific reason. PyTorch maintains its own CMake Modules and somebody should do the update of https://github.com/pytorch/pytorch/blob/main/cmake/Modules/FindBLAS.cmake